tests/mad_spark_alt_defaults_test.py

```
@@ -0,0 +1,123 @@
+"""
+Tests for mad_spark_alt command default values and validation.
+"""
+
+import subprocess
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)